src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="header_Logo">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav_list">
+          <li class="nav_list_li nav_list_li_first">
+            <a href="index.html" class="logo_link first_logo_link is-active">Apple</a>
+          </li>
+
+          <li class="nav_list_li">
+            <a href="index.html" class="logo_link">Samsung</a>
```

Comment:
Use anchor tag here. Clicking on it will be added to browser history, but the page will not needlessly reloaded.
```suggestion
            <a href="#" class="logo_link">Samsung</a>
```